src/microbe_stage/components/MicrobeColony.cs

```diff
@@ -891,26 +885,17 @@ public static void CalculateRotationMultiplier(this ref MicrobeColony colony, Ph
 
                 var distanceSquared = memberPosition.RelativePosition.LengthSquared();
 
-                if (distanceSquared < MathUtils.EPSILON)
-                    continue;
+                // Multiply both the propulsion and mass by the distance from center to simulate leverage
+                // This relies on the bounding of the cell rotation, as a colony can never be faster than the
+                // fastest cell inside it
+                var memberRotation = MicrobeInternalCalculations
+                        .CalculateRotationSpeed(colonyMember.Get<OrganelleContainer>().Organelles!)
```

Comment:
I was about to comment the following but then I read how you basically gutted the rotation speed calculation to no longer have any relation to the actual physics shape.
> This calculate rotation speed is going to absolutely kill, as this will compute the full physics shape for each colony member.

Reply:
That is correct. Any attempts to go half-and-half proved very cumbersome, so if we want to go with this PR's approach at all I think it's best to just stay away from the physics shapes as a whole. It's less accurate from a Newtonian standpoint, but it is easier to control for gameplay balance.

Reply:
I was kind of hoping that in the future we could go for full physics based rotating, which I guess now that I also think about it doesn't require the physics shape when calculating the allowed speed.

Still to clean up things the variant of the speed calculation taking in the physics body should be entirely removed and the code comments adjusted on where the speed values are calculated to say that they are only calculated there for historical reasons and don't currently require the physics state.